src/BusStop.js

```diff
@@ -0,0 +1,54 @@
+import { useState, useEffect } from 'react';
+
+function BusStop(props) {
+    const [actualDepartures, setActualDepartures] = useState([]);

```

Comment:
I think this variable name is ok, but just wanted to suggest using "liveDepartures" instead

Reply:
changed to "realTimeDepartures"